test/UtilsTest.js

```diff
@@ -244,212 +245,124 @@ describe('FlowLimit', async () => {
-            limit,
-            outAmount,
-            test.address,
-        ]);
-    });
-
-    it('Should revert after multiple ins and outs when final in flow exceeds limit', async () => {
-        await test.setFlowLimit(1).then((tx) => tx.wait);
-        await test.addFlowIn(1).then((tx) => tx.wait);
-        await test.addFlowOut(1).then((tx) => tx.wait);
-        await test.addFlowOut(1).then((tx) => tx.wait);
-
-        await expectRevert((gasOptions) => test.addFlowIn(2, gasOptions), test, 'FlowAmountExceededLimit', [1, 2, test.address]);
-    });
-
-    it('Should revert when flow + amount exceeds uint256 max', async () => {
-        await test.setFlowLimit(MaxUint256).then((tx) => tx.wait);
-        await test.addFlowIn(MaxUint256).then((tx) => tx.wait);
-
-        await expectRevert((gasOptions) => test.addFlowIn(1, gasOptions), test, 'FlowAmountOverflow', [1, MaxUint256, test.address]);
-    });
-
-    it('Should revert when accumulated out flow overflows', async () => {
-        const limit = MaxUint256.sub(1);
-
-        await test.setFlowLimit(limit).then((tx) => tx.wait);
-        await test.addFlowIn(1).then((tx) => tx.wait);
-        await test.addFlowOut(2).then((tx) => tx.wait);
-
-        await expectRevert((gasOptions) => test.addFlowOut(limit, gasOptions), test, 'FlowAmountOverflow', [limit, 2, test.address]);
-    });
-
-    it('Should revert after multiple ins and outs when final in overflows', async () => {
-        const limit = MaxUint256.sub(1);
-        const largeAmount = MaxUint256.sub(2);
-
-        await test.setFlowLimit(limit).then((tx) => tx.wait);
-        await test.addFlowIn(10).then((tx) => tx.wait);
-        await test.addFlowOut(1).then((tx) => tx.wait);
-
-        await expectRevert((gasOptions) => test.addFlowIn(largeAmount, gasOptions), test, 'FlowAmountOverflow', [
-            largeAmount,
-            10,
-            test.address,
-        ]);
-    });
-
-    it('Should revert in large in/out/in sequence that causes final in to overflow', async () => {
-        const half = MaxUint256.div(2);
-        const limit = half.add(2);
-
-        await test.setFlowLimit(limit).then((tx) => tx.wait);
-        await test.addFlowIn(half).then((tx) => tx.wait);
-        await test.addFlowOut(half).then((tx) => tx.wait);
-
-        await expectRevert((gasOptions) => test.addFlowIn(half.add(2), gasOptions), test, 'FlowAmountOverflow', [
-            half.add(2),
-            half,
-            test.address,
-        ]);
-    });
-
-    it('Should revert when total flow exceeds limit despite individual valid flows', async () => {
-        await test.setFlowLimit(1).then((tx) => tx.wait);
-        await test.addFlowIn(1).then((tx) => tx.wait);
-
-        await expectRevert((gasOptions) => test.addFlowIn(1, gasOptions), test, 'FlowLimitExceeded', [1, 2, test.address]);
-    });
+    const calculateExpectedArgs = async (error, isIn, amount, flowIn, flowOut, flowLimit, test) => {
```

Comment:
```suggestion
    const calculateExpectedArgs = async (error, flow, reverseFlow, flowAmount, flowLimit, test) => {
```
pass in flow/reverseFlow instead for simplicity?

Reply:
Updated code. I think passing `isInbound` makes the code easier to understand, especially since we're already using `const flow` in `getErrorArgs`. Otherwise, we're reusing the parameter name `flow` and then declaring `const flow` again inside the function, which could be confusing. Let me know what you think.